apps/trading/components/rewards-container/use-reward-row-data.spec.ts

```diff
@@ -0,0 +1,176 @@
+import type { AssetFieldsFragment } from '@vegaprotocol/assets';
+import { getRewards } from './use-reward-row-data';
+import * as Schema from '@vegaprotocol/types';
+
+const testData = {
+  rewards: [
+    {
+      rewardType: Schema.AccountType.ACCOUNT_TYPE_FEES_INFRASTRUCTURE,
+      assetId:
+        'ff3df1224e159f8619d9d66c56c21a11e10422dfa99dc45937fdff364de5e546',
```

Comment:
can we change these keys to something easier to read liket asset-1, asset-2 etc

Reply:
updated